app/models/Cookbooks.py

```diff
@@ -0,0 +1,20 @@
+import uuid
+from sqlmodel import Field, Relationship, SQLModel
```

Comment:
did you create a migration using alembic like these? https://github.com/the-olive-team/olive-api/tree/develop/app/alembic/versions

Reply:
see this on how to create the migration file https://alembic.sqlalchemy.org/en/latest/tutorial.html#create-a-migration-script